lib/api-group.js

```diff
@@ -23,6 +28,10 @@ class ApiGroup {
     this.requestOptions.cert = options.cert;
     this.requestOptions.key = options.key;
 
+    if (options.hasOwnProperty('insecureSkipTlsVerify')) {
```

Comment:
@bmeck is technically correct here (see: [`hasOwnProperty` vs `in`](http://stackoverflow.com/questions/13632999/if-key-in-object-or-ifobject-hasownpropertykey). I believe that the correct usage here would be:

``` js
if ('insecureSkipTlsVerify' in options) {
```